wolfssl/wolfcrypt/types.h

```diff
@@ -647,10 +648,11 @@ decouple library dependencies with standard string, memory and so on.
                 #include <ctype.h>
             #endif
 	    #if defined(HAVE_ECC) || defined(HAVE_OCSP) || \
-            defined(WOLFSSL_KEY_GEN) || !defined(NO_DSA)
+            defined(WOLFSSL_KEY_GEN) || !defined(NO_DSA) || \
+            defined(OPENSSL_EXTRA)
             #define XTOUPPER(c)     toupper((c))
-            #define XISALPHA(c)     isalpha((c))
         #endif
+        #define XISALNUM(c)     isalnum((c))
```

Comment:
Can we limit when this is compiled. Maybe only with OPENSSL_EXTRA. That might help later on with porting wolfSSL.